### DISCUSSION TITLE
How can I re-apply the education account?

### DISCUSSION THREAD

**Discussion by kaiyasi** [community/community#142694] [GitHub Education]
### Hi everyone,

I received the approved status, but while waiting, I removed the school account. Now, it seems to be stuck in an embarrassing state. 
How can I fix this?
![擷取](https://github.com/user-attachments/assets/a3bc50b4-15e1-4976-ae5c-8111387f5958)

**Comment by thomassiard** [comment]
Contact Support: Reach out to the platform’s support team to explain the situation. They can often restore your account or guide you on how to re-link it.

Reapply (if possible): Some platforms allow you to reapply with the same school credentials. Log into the education account page and check if there’s an option to re-verify your status.

Use the Approval Email: If you received an approval confirmation email, include it in your support request to speed up the verification process.

Clear Cache or Use a Different Browser: Sometimes, the system may get stuck due to cached data. Clear your browser’s cache or try logging in from a different device or browser to see if it helps.

**Reply by kaiyasi** [reply]
But how can i reach out to the platform's support team.
It seems they want user to use this system instead of reach them.

**Comment by davevad93** [comment]
Hi @mokoto0315 ,

It's not possible to re-apply once an application has been submitted and approved, and what exactly do you mean by “I have removed the school account”? Your status is `Congrats! Your request for @mokoto0315 was approved and your benefits are now available.` which means you should have full access to your `Student Developer Pack` offers, and I also note that you have the `PRO` badge.

If you can redeem any of the offers listed [here](https://education.github.com/pack), you have nothing to worry about.
